dspace/modules/journal-submit/src/main/java/org/datadryad/submission/DryadGmailService.java

```diff
@@ -46,7 +46,7 @@
      * @throws IOException
      */
     public DryadGmailService(File credentialPath, String clientsecretsFile, String scope, String user) {
-        myUserID = user;
```

Comment:
I think this is something that is needed when you re-authorize the user. I re-authorized the user on dev and everything is still fine.